azext_edge/edge/providers/adr/_help.py

```diff
@@ -835,6 +835,22 @@ def load_iotops_adr_help():
             az iot ops ns device endpoint inbound add opcua --device myDevice --instance myInstance -g myInstanceResourceGroup --name myOPCUAEndpoint --endpoint-address "opc.tcp://192.168.1.100:4840" --security-policy "Basic256Sha256" --security-mode "SignAndEncrypt" --run-asset-discovery
     """
 
+    helps[
+        "iot ops ns device endpoint inbound add rest"
+    ] = """
+        type: command
+        short-summary: Add a rest inbound endpoint to a device in a Device Registry namespace.
+
+        examples:
+        - name: Add a basic rest endpoint to a device
+          text: >
+            az iot ops ns device endpoint inbound add rest --device myDevice --instance myInstance -g myInstanceResourceGroup --name myEndpoint --endpoint-address "https://api.example.com/data"
+
+        - name: Add a rest endpoint with authentication
+          text: >
+            az iot ops ns device endpoint inbound add rest --device myDevice --instance myInstance -g myInstanceResourceGroup --name myEndpoint --endpoint-address "https://api.example.com/data" --user-ref "secretRef:username" --pass-ref "secretRef:password"
```

Comment:
Where does this input format come from `secretRef:username` is it actually required by ADR? It's different than other secretRef values for other services.

Reply:
are you talking about `--custom-x509-secret-ref`?

<img width="995" height="96" alt="image" src="https://github.com/user-attachments/assets/7d09531c-98eb-4355-8b13-e746c85b59a4" />

if so - I will move this comment to the overall adr pr since I have another branch focusing on help + param + other changes that aren't adding or changing command structures